src/cdk/tree/nested-node.ts

```diff
@@ -60,6 +58,14 @@ export class CdkNestedTreeNode<T> extends CdkTreeNode<T> implements AfterContent
               protected _tree: CdkTree<T>,
               protected _differs: IterableDiffers) {
     super(_elementRef, _tree);
+    // The classes are directly added here instead of in the host property because classes on
+    // the host property are not inherited with View Engine.
+    this._elementRef.nativeElement.classList.add('cdk-tree-node');
```

Comment:
I don't think you need to add `cdk-tree-node` because its already added by the super-class constructor.